core/modules/config/config.install

```diff
@@ -0,0 +1,11 @@
+<?php
+
+/**
+ * Implements hook_install().
+ */
+function config_install() {
```

Comment:
I'm not sure this is needed. Config module is optional (though it is installed in the standard profile). It doesn't seem like enabling the module should fire this. It's also a bit odd that it would be triggered on new installs of Backdrop when Config module is first installed, but because it's installed in order, only some of the core modules would have their config files updated.